exporter/datadogexporter/integrationtest/integration_test.go

```diff
@@ -304,3 +304,269 @@ func getGzipReader(t *testing.T, reqBytes []byte) io.Reader {
 	require.NoError(t, err)
 	return reader
 }
+
+func getIntegrationComputeTopLevelBySpanKindTestCollector(t *testing.T, url string, factories otelcol.Factories) (*otelcol.Collector, string) {
```

Comment:
nit: you can have `getIntegrationTestCollector` accept the config string instead of making a separate function.